fleetshard/pkg/central/reconciler/reconciler.go

```diff
@@ -40,6 +40,7 @@ const (
 	helmReleaseName = "tenant-resources"
 
 	managedServicesAnnotation = "platform.stackrox.io/managed-services"
+	tenantIDLabelKey          = "rhacs.redhat.com/tenant"
```

Comment:
Why label and not annotation?

Reply:
I think it fits a bit better semantically because it identifies the resources. It's also more useful than an annotation because you can do something like `kubectl get pods -l tenant=xxx`. Not a big deal, but why not use it.

Reply:
+1 for label. Labels are used for identification, whereas annotations define different meta data and sometimes modify a pods behaviour (i.e. a controller).


Suggestion: being more explicit in the annotation key:

```suggestion
	tenantIDLabelKey          = "rhacs.redhat.com/tenant-id"
```

Reply:
Well, ok, I can see a potential use case to find pods in multi-tenant setup one day.
This ID is going to modify central pods behavior. I'll need to update the backend, once we agree on the naming.
@SimonBaeumer, please expand how the similar namespace label is used, why cannot the name be the same?

Reply:
But you don't have to specify the value to query for the existence of the label.

Reply:
>But you don't have to specify the value to query for the existence of the label.

That's correct.

@stehessel Are you worried about the number of labels on an k8s object? 🤔

Reply:
In this case yes (well not really worried, just a bit baffled). I see no reason for two tenant identifier labels.

Reply:
Agree, let's use `rhacs.redhat.com/tenant` then.